tests/t0112_gateway_cors_test.go

```diff
@@ -0,0 +1,52 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/ipfs/gateway-conformance/tooling/test"
+	. "github.com/ipfs/gateway-conformance/tooling/test"
+)
+
+func TestCors(t *testing.T) {
+	cidHello := "bafkqabtimvwgy3yk" // hello
+
+	tests := SugarTests{
+		{
+			Name: "GET Response from Gateway should contain CORS headers",
```

Comment:
I found myself explaining many times why we do this, let's elaborate bit more while we are at it, so people reading tests will also learn rationale:

```suggestion
			Name: "GET Responses from Gateway should include CORS headers allowing JS from other origins to read the data cross-origin.",
```